hackIDE_project/settings.py

```diff
@@ -23,11 +23,10 @@
 SECRET_KEY = '+h*i@$52+w(_e#etvzgnkjq!q0ajz1qpgs-y9%89x4w3nlct=m'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-#(os.environ.get('HACKIDE_DEBUG') != None)
+DEBUG = (os.environ.get('HACKIDE_DEBUG') != None)
```

Comment:
In case you don't understand this, here's the explanation - 

`HACKIDE_DEBUG` is to be set to `True` on your local machine as an environment variable. So for your local machine, `os.environ.get('HACKIDE_DEBUG')` won't be `None`. Hence, `DEBUG = True` which is desired on your local machine.

Read more [here](https://github.com/sahildua2305/hackIDE#how-to-run-the-server-locally).